docs/troubleshooting.md

```diff
@@ -17,14 +17,18 @@ the `--log-level` flag to the `otelcol` process. See `--help` for more details.
 $ otelcol --log-level DEBUG
 ```
 
+Note that debug logs do not provide a lot of additional value today. Instead,
```

Comment:
I am not sure why we say so. Debug logs are used extensively by many components. Perhaps we shouldn't recommend "instead", we should recommend "in addition"?

Reply:
Perhaps true, but in practice not much is / nothing really useful is emitted today. Try it with core -- the only thing I can get to emit debug logs is the memorylimiter for currently used memory. No different on contrib. Changing this toggle will not be able to help the common use debug/troubleshoot anything (while the logging exporter definitely will).

Reply:
I agree with Tigran here. Maybe debug logs are not very useful in core today but if this applies to contrib or other distros it's likely that debug logs are useful for at least some components.

Reply:
It is true that there is not a ton of debug logs emitted when everything is working fine. However, I do expect to see some debug longs when errors happen. There are 176 hits for `.Debug(`, so I hope at least some are useful.
Not a big deal, but I did not want it look like we are discouraging using logs for debugging purposes.